src/VisualStudio/Core/Def/Implementation/Remote/GlobalNotificationRemoteDeliveryService.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Notification;
+using Microsoft.CodeAnalysis.Remote;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Remote
+{
+    /// <summary>
+    /// Delivers global notifications to remote services.
+    /// </summary>
+    internal sealed class GlobalNotificationRemoteDeliveryService : IDisposable
+    {
+        private enum GlobalNotificationState
+        {
+            NotStarted,
+            Started,
+            Finished
```

Comment:
i don't see the Finished value ever used.  did i miss it?

Reply:
I see, it used to be used. I deleted that code :) Will remove this value.

---
In reply to: [419027233](https://github.com/dotnet/roslyn/pull/43928#discussion_r419027233) [](ancestors = 419027233)